scripts/release/theme-extractor.mjs

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env node
-const fs = require('fs');
-const path = require('path');
-const fg = require('fast-glob');
+import fs from 'fs';
+import path from 'path';
+import fg from 'fast-glob';
 
-const { log, errorHandler, success, ROOT } = require('../helpers');
-const { ELEMENT_DIST, PACKAGE_ROOT, getElementList, getElementTagName } = require('./util');
+import { log, errorHandler, success, ROOT } from '../helpers/index.mjs';
+import { ELEMENT_DIST, PACKAGE_ROOT, getElementList, getElementTagName } from './util.cjs';
 
 // List of themes to be extracted
 const THEMES = ['halo', 'solar'];
 
 // Element package scope
-const PACKAGE_NAME = require(`${PACKAGE_ROOT}/package.json`).name;
+const PACKAGE_NAME = JSON.parse(
```

Comment:
Let's refactor this JSON paraser as a helper.

Reply:
Refactored